src/python/grpcio/README.rst

```diff
@@ -51,7 +51,7 @@ package named :code:`python-dev`).
   $ git submodule update --init
 
   # For the next two commands do `sudo pip install` if you get permission-denied errors
-  $ pip install -rrequirements.txt
+  $ pip install -r requirements.txt
```

Comment:
no, this is not a typo. please revert this line completely.
https://github.com/grpc/grpc/blob/d64fd75dd889edc41049a87d2f4345be689a9936/tools/run_tests/artifacts/build_artifact_python.sh#L97